src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerOptions.cs

```diff
@@ -29,6 +29,11 @@ public class KestrelServerOptions
         public bool AddServerHeader { get; set; } = true;
 
         /// <summary>
+        /// Gets or sets a value that determines if Kestrel should use the transport thread thread when executing user code.
```

Comment:
Now this isn't pubternal, can we try scaring people away from using this in the doc comments?